src/ansys/dpf/post/selection.py

```diff
@@ -0,0 +1,320 @@
+"""
+Selection
+---------
+
+"""
+
+from ansys.dpf.core import Scoping, time_freq_scoping_factory, Field, locations, operators, \
+    Workflow, types
+from ansys.dpf.core.field import _get_size_of_list
+from typing import Union
+
+
+class _WfNames:
+    data_sources = "data_sources"
+    scoping = "scoping"
+    scoping_a = "scoping_a"
+    scoping_b = "scoping_b"
+    streams = "streams"
+
+
+class Selection:
+    """The ``Selection`` class helps defining the domain on which results are evaluated.
+    The results domain defines the time frequency and the spatial selection.
+
+    Parameters
+    ----------
+
+
+    """
+
+    def __init__(self, server=None):
+        self._server = server
+        self._spatial_selection = SpatialSelection(server=server)
+        self._time_freq_selection = TimeFreqSelection(server=server)
+        self._qualifier_selection = None
```

Comment:
I think we should be instantiating the object the properties itself. Like

```suggestion
    def __init__(self, time_freq_select=None, nodes=Node, server=None):
        self._server = server
        self._spatial_selection = SpatialSelection(nodes=nodes, server=server)
        self._time_freq_selection = TimeFreqSelection(time_freq_select=time_freq_select, server=server)
        self._qualifier_selection = None
```